markup/goldmark/goldmark_config/config.go

```diff
@@ -44,9 +44,10 @@ var Default = Config{
 		LinkifyProtocol: "https",
 		TaskList:        true,
 		CJK: CJK{
-			Enable:              false,
-			EastAsianLineBreaks: false,
-			EscapedSpace:        false,
+			Enable:                   false,
+			EastAsianLineBreaks:      false,
+			EastAsianLineBreaksStyle: "simple",
```

Comment:
I placed the style specification settings at this level to maintain backward compatibility.

`"simple"` is the current newline style. If not specified, it defaults to `"simple"`.
Another option available is `"css3draft"`.